lib/transformer.dart

```diff
@@ -258,21 +256,23 @@ class _BootstrapFileBuilder {
               '(possibly transitive).');
         }
       }
+
       readSuperClassAnnotations(clazz.supertype);
       _readAnnotations(clazz);
     }
   }
 
   bool _readAnnotations(Element element) {
     var found = false;
-    if (element.metadata.isEmpty) return found;
+    // analyzer 0.29 doesn't allow this optimization :
+    //if (element.metadata.isEmpty) return found;
```

Comment:
I've found not an easy way to reproduce the issue, sorry. All that I can say is that running `initialize` on a project using a lot of polymer components most of the `HtmlImport` where catched but some where not. In those cases even if `element.metadata` of the corrsponding `LibraryElement` was empty, the `element.computeNode()` will return the actual annotaions.

Couldn't find a pattern though. And even if I made a tiny project with only those components that were skipped then in that tiny project they were processed !

So couldn't be a matter of "lazy" loading or something like that ?